package.json

```
@@ -18,7 +18,12 @@
     "test:watch": "jest --watch",
     "test:cov": "jest --coverage",
     "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
-    "test:e2e": "jest --config ./test/jest-e2e.json"
+    "test:e2e": "jest --config ./test/jest-e2e.json",
+    "typeorm": "ts-node ./node_modules/typeorm/cli",
+    "typeorm:run": "npm run typeorm migration:run -- -d src/ormconfig.ts",
+    "typeorm:generate": "npm run typeorm -- -d ./src/ormconfig.ts migration:generate ./src/migrations/$npm_config_name",
+    "typeorm:create": "npm run typeorm -- migration:create ./src/migrations/$npm_config_name",
```

ти бачив, що назва міграції створюється некоректна, тому попробуй по іншому, такоє генерувати міграції ми не будемо, як я казав, потрібно писати вручну
 
 ```suggestion
    "typeorm:generate": "npm run typeorm -- -d ./src/ormconfig.ts migration:generate ./src/migrations/$npm_config_name",
    "typeorm:create": "npm run typeorm -- migration:create  \“src/migrations/$npm_config_name\“",
```

---

Скільки варінтів не пробував, все одно назву не підтягує(